wp-includes/widgets/class-wp-widget-media-audio.php

```diff
@@ -151,7 +151,7 @@ public function enqueue_admin_scripts() {
 
 		$exported_schema = array();
 		foreach ( $this->get_instance_schema() as $field => $field_schema ) {
-			$exported_schema[ $field ] = wp_array_slice_assoc( $field_schema, array( 'type', 'default', 'enum', 'minimum', 'format', 'media_prop' ) );
+			$exported_schema[ $field ] = wp_array_slice_assoc( $field_schema, array( 'type', 'default', 'enum', 'minimum', 'format', 'media_prop', 'should_preview_update' ) );
```

Comment:
Thanks for catching this... do you think we should have a base array of exported fields that should be included on each class?  I suppose not that critical but just a thought.